deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -1177,6 +1177,7 @@ def get(self, request, name, stage):
             "current_image": current_image,
             "golden_image": golden_image,
             "base_images_events": base_images_update_events,
+            "current_cluster": json.dumps(current_cluster),
```

Comment:
just curious why json.dumps is needed?

Reply:
yea, it is needed. otherwise the content includes u' before all fields.